src/mono/sample/iOS-NativeAOT/Program.csproj

```
@@ -4,6 +4,7 @@
     <OutputPath>bin</OutputPath>
     <IntermediateOutputPath>$(MSBuildThisFileDirectory)/obj/</IntermediateOutputPath>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <HybridGlobalization>true</HybridGlobalization> 
```

It should be redundant, as it's set by default now. In case you wanted to keep the sample in the same shape as it was before,
```suggestion
    <HybridGlobalization>false</HybridGlobalization> 
```

---

I'm not sure I follow, where it is set by default?

---

we only support Hybrid now so we don't need to explicitly set it to true.

---

After having this line https://github.com/dotnet/runtime/pull/93220/files#diff-2884e8c39a88d26cb57ef06adca164f61238c4ad1c593d8e38bcb167212faa38R17 yes these changes are redundant, will update them.